ui-v2/src/components/ui/docs-link.tsx

```diff
@@ -17,13 +19,14 @@ type DocsID = keyof typeof DOCS_LINKS;
 
 type Props = {
 	id: DocsID;
+	label?: "View Docs" | "Documentation";
```

Comment:
Need to make this update. Seems like docs link can either be `View Docs` or `Documentation`